src/System.Memory/src/System/ReadOnlySpan.Portable.cs

```diff
@@ -131,6 +131,24 @@ internal ReadOnlySpan(Pinnable<T> pinnable, IntPtr byteOffset, int length)
             }
         }
 
+        /// <summary>
+        /// Returns a reference to the 0th element of the Span. If the Span is empty, returns null reference.
+        /// It can be used for pinning and is required to support the use of span within a fixed statement.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public unsafe ref readonly T GetPinnableReference()
+        {
+            if (_length != 0)
+            {
+                if (_pinnable == null)
+                {
+                    unsafe { return ref Unsafe.AsRef<T>(_byteOffset.ToPointer()); }
```

Comment:
Why is unsafe needed here when the whole method is marked unsafe?

Reply:
Good point (copy/paste error)